src/JsonReader.cs

```diff
@@ -533,6 +533,28 @@ sealed class NonProperty : IJsonProperty<Unit, JsonReadResult<Unit>>
                NonProperty.Instance,
                (v, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _) => v);
 
+    struct ObjectValueState<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10, T11, T12, T13, T14, T15, T16>
```

Comment:
nit: Let's call this `ObjectReadState`

Reply:
Done in b268b7dfcc91583041a741116f63703973f0b5ad